components/usage/pkg/stripe/stripe_test.go

```diff
@@ -83,3 +85,25 @@ func TestCustomerQueriesForTeamIds_MultipleQueries(t *testing.T) {
 		})
 	}
 }
+
+func TestGetUpcomingInvoice(t *testing.T) {
+	secretKey := os.Getenv("TEST_KEY_STRIPE")
+	teamId := os.Getenv("TEST_TEAM_ID")
```

Comment:
How are these injected during test runs on CI?

Reply:
If it's not gonna run most of the time on CI, does it make sense to keep it here?

Reply:
removing...